static/docs/commands-reference/version.md

```diff
@@ -0,0 +1,41 @@
+# version
+
+This command shows the system/environment information along with the DVC version.
+
+## Synopsis
+
+```usage
+    usage: dvc version [-h] [-q | -v]
+```
+
+## Description
+
+Running the command `dvc version` outputs the
+following information about the system/environment:
+
+Type | Detail
+---- | ------
+`DVC version` | Version of DVC along with information about the project's Git repository
+`Python version` | Version of the Python being used for the project in which DVC is initialized
+`Platform` | Information about the operating system of the machine
+
+## Options
+
+* `-h`, `--help` - prints the usage/help message, and exit.
+
+* `-q`, `--quiet` - does not write anything to standard output. Exit with 0 if
+  no problems arise, otherwise 1.
+
+* `-v`, `--verbose` - displays detailed tracing information.
+
+## Example
+
+Getting the DVC version and environment information:
+
+```dvc
+    $ dvc version
+    
```

Comment:
remove trailing whitespaces please

Reply:
@shcheklein Is there a VSCode extension for this as well?